scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -0,0 +1,103 @@
+# Quickstart - Joint Calling with the Broad Genomic Variant Store 
+
+**Note** The markdown source for this quickstart is maintained in the the  [GATK GitHub Repository](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/TERRA_QUICKSTART.md).  Submit any feedback, corrections or improvements in a pull request there.  Do not edit this file directly.
+
+## Overview
+Through this QuickStart you will learn how to use the Broad Genomic Variant Store to create a GATK VQSR Filtered joint callset VCF for whole genome samples.
+
+The sequencing data in this quickstart came from the [AnVIL 1000G High Coverage workspace](https://app.terra.bio/#workspaces/anvil-datastorage/1000G-high-coverage-2019)
+
+**Note:** VQSR dies with the default/recommended configuration, so we set SNP max-gaussians to 4 here...
+
+## Prerequisites
+
+This quickstart assumes that you are familiar with Terra workspaces, the data model and providing input parameters and launching workflows.
+
+1. You will need to have or create a BigQuery dataset (we'll call this `datasetname` later on). 
```

Comment:
Is this because you need to be an owner so that you can grant these IAM roles?  Or do you need to be owner for some other reason (ie say someone else granted you these roles, would that work?)

Reply:
Also -- the directions didn't really talk about making a temp dataset... was that a problem?  did you have to make one @rsasch ?

Reply:
> Is this because you need to be an owner so that you can grant these IAM roles? Or do you need to be owner for some other reason (ie say someone else granted you these roles, would that work?)

Yes, I needed to be "Owner" to add those roles for my proxy group.  I didn't test a scenario where those roles were added by someone else, but I don't see why that wouldn't work, as long as I was still able to do things that cost money in the specified Google project.

Reply:
> Also -- the directions didn't really talk about making a temp dataset... was that a problem? did you have to make one @rsasch ?

I think I might have mis-read my notes.  It looks like I ran into the issue in the CreateVetTables call: “BigQuery error in mk operation: Access Denied: Dataset”, but it worked once I used “BigQuery Data Editor” instead.